lib/workers/repository/dependency-dashboard.ts

```diff
@@ -6,6 +6,53 @@ import { getProblems, logger } from '../../logger';
 import { platform } from '../../platform';
 import { BranchConfig, BranchResult } from '../types';
 
+export interface DependencyDashboard {
+  dependencyDashboardChecks: Record<string, string>;
+  dependencyDashboardRebaseAllOpen: boolean;
+}
+
+export function parseDashboardIssue(issueBody: string): DependencyDashboard {
```

Comment:
```suggestion
function parseDashboardIssue(issueBody: string): DependencyDashboard {
```

Do not export because it's only used in this file?